packages/react-components/react-breadcrumb-preview/stories/BreadcrumbButton/BreadcrumbButtonWithHrefAttribute.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { Breadcrumb, BreadcrumbItem, BreadcrumbButton, BreadcrumbDivider } from '@fluentui/react-breadcrumb-preview';
+import { bundleIcon, CalendarMonth20Filled, CalendarMonth20Regular, GridDots20Regular } from '@fluentui/react-icons';
+import { ButtonProps } from '@fluentui/react-components';
+
+const CalendarMonth = bundleIcon(CalendarMonth20Filled, CalendarMonth20Regular);
+
+type Item = {
+  key: number;
+  item?: string;
+  linkProps?: {
+    'aria-label'?: string;
+    href?: string;
+    icon?: ButtonProps['icon'];
+    disabled?: boolean;
+  };
+};
+
+const linkItems: Item[] = [
+  {
+    key: 0,
+    item: 'Default',
+    linkProps: {
+      href: 'https://react.fluentui.dev/?path=/docs/preview-components-breadcrumb--default',
+    },
+  },
+  {
+    key: 1,
+    item: 'Size',
+    linkProps: {
+      icon: <CalendarMonth />,
+      'aria-label': 'Item 1',
+      href: 'https://react.fluentui.dev/?path=/docs/preview-components-breadcrumb--breadcrumb-size',
+      disabled: true,
+    },
+  },
+  {
+    key: 2,
+    item: 'Overflow',
+    linkProps: {
+      icon: <GridDots20Regular />,
+      href: 'https://react.fluentui.dev/?path=/docs/preview-components-breadcrumb--breadcrumb-with-overflow',
+    },
+  },
+  {
+    key: 3,
+    item: 'Breadcrumb with Tooltip',
+    linkProps: {
+      disabled: true,
+      href: 'https://react.fluentui.dev/?path=/docs/preview-components-breadcrumb--breadcrumb-with-tooltip',
+    },
+  },
+  {
+    key: 4,
+    item: 'Breadcrumb focus mode',
+    linkProps: {
+      href: 'https://react.fluentui.dev/?path=/docs/preview-components-breadcrumb--breadcrumb-focus-mode',
+    },
+  },
+];
+
+function renderLink(el: Item, isLastItem: boolean = false) {
+  return (
+    <React.Fragment key={`${el.key}-link`}>
+      <BreadcrumbItem>
+        <BreadcrumbButton {...el.linkProps} current={isLastItem} as="a">
+          {el.item}
+        </BreadcrumbButton>
+      </BreadcrumbItem>
+      {!isLastItem && <BreadcrumbDivider />}
+    </React.Fragment>
+  );
```

Comment:
Based on Ling's comment https://github.com/microsoft/fluentui/pull/29302#discussion_r1340972904, the last item shouldn't be interactive, right? E.g.:

```suggestion
  if (!isLastItem) {
    return (
      <React.Fragment key={`${el.key}-link`}>
        <BreadcrumbItem>
          <BreadcrumbButton {...el.linkProps} as="a">
            {el.item}
          </BreadcrumbButton>
        </BreadcrumbItem>
        <BreadcrumbDivider />
      </React.Fragment>
    );
  } else {
    return <BreadcrumbItem current={true} key={el.key}>{el.item}</BreadcrumbItem>;
  }
```

Reply:
BreadcrumbButton is done the way so last item is not interactive anyway. So this can be done both ways. The only situation when BreadcrumbButton has to be used is for `small` size because of request from the designers to change spacing for the small items. It's not the best solution we have now but it's because this change request was made after spec review